products/uosv20/transforms/constants.xslt

```diff
@@ -0,0 +1,13 @@
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<xsl:include href="../../shared/transforms/shared_constants.xslt"/>
+
+<xsl:variable name="product_long_name">Uniontech OS V20</xsl:variable>
+<xsl:variable name="product_short_name">Uos v20</xsl:variable>
+<xsl:variable name="product_stig_id_name">UOSV20_STIG</xsl:variable>
+<xsl:variable name="prod_type">uos</xsl:variable>
+
+<!-- Define URI of official Center for Internet Security Benchmark for Uniontech OS -->
+<xsl:variable name="cisuri">https://www.cisecurity.org/benchmark/centos_linux</xsl:variable>
```

Comment:
Please remove this line. We can add it back with the correct URL once the OS has a CIS benchmark.